tcms/core/middleware.py

```diff
@@ -32,10 +40,13 @@ def process_request(self, request):
                     }
                 ),
             )
+        return None
 
 
 class CheckUnappliedMigrationsMiddleware(MiddlewareMixin):
     def process_request(self, request):
+        if "setup" in request.path:
+            return
```

Comment:
I think a better match would be `/setup/ == request.path` or similar. You want to match this exactly, not loosely.